src/controllers/product-controller.js

```diff
@@ -1,4 +1,79 @@
-// eslint-disable-next-line no-unused-vars
+const config = require('config');
+const FileType = require('file-type');
+const Promise = require('bluebird');
+
 const { ProductModel } = require('../models');
+const S3Lib = require('../libs/s3-lib');
+const NotFoundError = require('../errors/not-found-error');
+const ForbiddenError = require('../errors/forbidden-error');
+
+
+async function uploadImages(req, res, next) {
+  try {
+    const { accessKeyId, secretAccessKey, bucketName } = config.get('aws');
```

Comment:
define these values on the file start, after requires block, as they will be the same for each uploadImages() call, to not load them into memory on each function call then clean the memory